validator/accounts/account.go

```diff
@@ -136,16 +136,13 @@ func Exists(keystorePath string) (bool, error) {
 // CreateValidatorAccount creates a validator account from the given cli context.
 func CreateValidatorAccount(path string, passphrase string) (string, string, error) {
 	if passphrase == "" {
-		log.Info("Create a new validator account for eth2")
-		log.Info("Enter a password:")
-		bytePassword, err := terminal.ReadPassword(int(os.Stdin.Fd()))
+		log.Info("Creating a new validator account for eth2")
+		enteredPassphrase, err := cmd.EnterPassword()
 		if err != nil {
-			log.Fatalf("Could not read account password: %v", err)
-			return path, passphrase, err
+			log.Fatal(err)
+			return path, enteredPassphrase, err
```

Comment:
no point of returning the result here, when you have a fatal log above

Reply:
Yeah this is duplicated several times in the validator client, was gonna make another PR making sure it's either handled or returned, never both. Is that fine?